fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
Ok, let's move forward with this PR and I'll add that to fcrepo-camel on Monday (this is exactly why I wanted to wait on the fcrepo-camel/4.5.0 release). We can track this issue here: https://jira.duraspace.org/browse/FCREPO-2345